File: DevExpress-Demo/CS/RibbonDemo.Wpf/Modules/RibbonSimplePad/ViewModels/RecentItemViewModel.cs
using System;

namespace RibbonDemo {
    public class RecentItemViewModel {
        static int count = 0;
        public string Caption { get; set; }
        public string Description { get; set; }
        public DateTime DateModified { get; set; }
        public RecentItemViewModel() { }
        public RecentItemViewModel(string caption, string description) {
            Caption = caption;
            Description = description;
            DateModified = DateTime.Now.AddDays(count);
            count++;
        }
    }
}

File: DevExpress-Demo/VB/RibbonDemo.Wpf/Modules/RibbonSimplePad/ViewModels/RecentItemViewModel.vb
Namespace RibbonDemo

    Public Class RecentItemViewModel

        Private Shared count As Integer = 0

        Public Property Caption As String

        Public Property Description As String

        Public Property DateModified As Date

        Public Sub New()
        End Sub

        Public Sub New(ByVal caption As String, ByVal description As String)
            Me.Caption = caption
            Me.Description = description
            DateModified = Date.Now.AddDays(count)
            count += 1
        End Sub
    End Class
End Namespace
